projects/packages/jetpack-mu-wpcom/composer.json

```diff
@@ -4,7 +4,7 @@
 	"type": "jetpack-library",
 	"license": "GPL-2.0-or-later",
 	"require": {
-		"php": ">=7.2",
```

Comment:
This change should be outside of this PR.

Reply:
Done in https://github.com/Automattic/jetpack/pull/41928 and https://github.com/Automattic/jetpack/pull/41931